dfx.json

```diff
@@ -2,11 +2,19 @@
 	"dfx": "0.20.0",
 	"canisters": {
 		"signer": {
-			"candid": "src/backend/backend.did",
-			"package": "backend",
-			"type": "rust",
-			"optimize": "cycles",
-			"gzip": true
+			"type": "custom",
+			"candid": "https://raw.githubusercontent.com/dfinity/chain-fusion-signer/v0.1.1/src/signer/signer.did",
+			"wasm": "https://github.com/dfinity/chain-fusion-signer/releases/download/v0.1.1/signer.wasm.gz",
+			"init_arg": "(variant { Init = record { ecdsa_key_name = \"key_1\"; ic_root_key_der = null;}})",
```

Comment:
`"key_1"` is only for local development, right? And is `ic_root_key_der` different for local?

Reply:
key_1 is the prod value.  I was hoping that we could use only prod values here.  I think it's unclear whether that would work so I'll update the PR to use the same keys as before.

Reply:
Done.  Please see `scripts/build.signer.sh`